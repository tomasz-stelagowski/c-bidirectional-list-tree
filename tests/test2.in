ADD_NODE 0
ADD_NODE 1
ADD_NODE 1
ADD_NODE 3
ADD_NODE 3
ADD_NODE 0
RIGHTMOST_CHILD 0
ADD_NODE 1
DELETE_NODE 7
RIGHTMOST_CHILD 1
DELETE_NODE 3
RIGHTMOST_CHILD 1
ADD_NODE 0
ADD_NODE 1
ADD_NODE 2
ADD_NODE 0
DELETE_SUBTREE 11
ADD_NODE 1
RIGHTMOST_CHILD 9
ADD_NODE 1
DELETE_NODE 13
RIGHTMOST_CHILD 2
DELETE_NODE 1
DELETE_NODE 6
RIGHTMOST_CHILD 0
DELETE_NODE 8
RIGHTMOST_CHILD 0
DELETE_NODE 12
RIGHTMOST_CHILD 0
DELETE_NODE 9
RIGHTMOST_CHILD 0
DELETE_NODE 2
RIGHTMOST_CHILD 10
DELETE_NODE 4
DELETE_NODE 5
RIGHTMOST_CHILD 0
