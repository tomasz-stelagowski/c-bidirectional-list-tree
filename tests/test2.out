OK
OK
OK
OK
OK
OK
6
OK
OK
3
OK
5
OK
OK
OK
OK
OK
OK
-1
OK
OK
10
OK
OK
8
OK
12
OK
9
OK
5
OK
-1
OK
OK
10
